BIBLIOGRAPHY.md

```diff
@@ -0,0 +1,30 @@
+This file contains a list of papers in chronological order that have been published 
+using Mozilla's DeepSpeech.
```

Comment:
nit: bibtex entry to cite deepspeech?

Reply:
@lissyx We don't have a DeepSpeech paper yet! When the CommonVoice paper is published we can cite that too.

Reply:
That might still be useful for people to refer to us

Reply:
Should we include the Common Voice preprint here already and then update it once it's published?

Reply:
I think that's a good idea, I've added a section for "To appear" where we can put submitted/accepted but not yet published papers.